app/questionnaire/questionnaire_schema.py

```diff
@@ -95,11 +95,15 @@ def get_default_answer(self, answer_id):
         except KeyError:
             question = block['question_variants'][0]['question']
 
+        answer_schema = None
         for this_answer in question['answers']:
             if this_answer['id'] == answer_id:
                 answer_schema = this_answer
                 break
 
+        if not answer_schema:
+            return None
```

Comment:
Since there are already multiple `try excepts` you could just add `NameError` into the except below instead of the `if`.
Thoughts?

Reply:
I prefer the try/except :+1:

Reply:
Ya, that's why I am saying we could remove the `if` and use the already defined try/except?

Reply:
We could remove all of this and get the answer by id from `self._answers_by_id`

Reply:
This is to get the schema default though?

Reply:
garh that's what you get for copy pasting form a different module without thinking :man_facepalming: